docs/concepts/Override/README.md

```diff
@@ -0,0 +1,275 @@
+# Override
+
+## Overview
+The `ClusterResourceOverride` and `ResourceOverride` provides a way to customize resource configurations before they are propagated 
+to the target cluster by the `ClusterResourcePlacement`.
+
+## Difference Between `ClusterResourceOverride` And `ResourceOverride`
+
+`ClusterResourceOverride` represents the cluster-wide policy that overrides the cluster scoped resources to one or more
+clusters while `ResourceOverride` will apply to resources in the same namespace as the namespace-wide policy.
+
+> **Note:** If a namespace is selected by the `ClusterResourceOverride`, ALL the resources under the namespace are selected
+automatically.
+
+If the resource is selected by both `ClusterResourceOverride` and `ResourceOverride`, the `ResourceOverride` will win

```

Comment:
```suggestion
If the resource is selected by both `ClusterResourceOverride` and `ResourceOverride`, precedence is granted to the `ResourceOverride` when resolving conflicts.
```

Reply:
gpt also suggested using `precedence`.
my concern is if it will confuse people in another way? we apply the resourceOverride first and then ClusterResourceOverride.